src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1382,6 +1382,31 @@ More over, the native Python Script Engine depends on the py4j library in order
 Further information can be found https://www.py4j.org/install.html[in the official py4j documentation^].
 
 
+== Full control of the scheduling policy
+
+Default scheduling policy can be extended to perfectly fit with user scheduling constraints. For instance, we integrated a license policy,
```

Comment:
I'd remove the comma after `a license policy`. So it would be:

> For instance, we integrated a license policy to control the number of concurrent licenses usage.

What do you think ?

Reply:
yes